tests/behat/theme_boost_union_feelsettings_navigation.feature

```diff
@@ -187,6 +187,34 @@ Feature: Configuring the theme_boost_union plugin for the "Navigation" tab on th
       | yes     | should      |
       | no      | should not  |
 
+  Scenario: Setting: Course category breadcrumbs (verify that course sections are properly displayed)
+    Given the following "categories" exist:
+      | name           | category | idnumber | category |
+      | Category E     | 0        | CE       | 0        |
+      | Category ED    | 1        | CED      | CE       |
+    And the following "courses" exist:
+      | fullname  | shortname | category |
+      | Course C1 | CC1       | CE       |
+      | Course C2 | CC2       | CED      |
+    And the following "course enrolments" exist:
+      | user     | course | role           |
+      | teacher1 | CC2    | editingteacher |
+    And the following config values are set as admin:
+      | config              | value     | plugin            |
+      | categorybreadcrumbs | yes       | theme_boost_union |
+    And "Course C1" has been set to one page per section
+    And "Course C2" has been set to one page per section
+    When I log in as "teacher1"
+    And I am on section "1" page of "Course C1" course
+    Then "Category E" "link" should exist in the ".breadcrumb" "css_element"
+    And "Topic 1" "link" should not exist in the ".breadcrumb" "css_element"
```

Comment:
I am wondering why you are testing this case for Course C1. The teacher is not enrolled into the course, so he will end up on the enrolment page and not on the section page. On this page, the topic link is not added to the breadcrumb, that's right. But if this was really your intention, then we should add a comment about this fact to this line.

On the other hand, if this was not your intention and the teacher should have been enrolled into Course C1, then this line would fail as the topic link is displayed in the breadcrumb.

Please clarify :)

Reply:
Okay, I've looked at the code again and tried to remember...

First, I looked what a user sees when he is not enrolled (without the patch):
![image](https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/assets/10991509/b15e4245-9b81-416c-bda4-50477c72fbab)

Independent of my reasoning for my PR I'd say that logically, the categories should come first, then "Einschreibeoptionen". Okay, I'd even say that before "Einschreibeoptionen" should maybe be a link to the course itself, but I'll be not picky.

But this doesn't in any way explain why I did the behat test that way. What I probably meant was to test the way the breadcrumb looks if you can view the section without being enrolled. So either as an admin or as someone with the right permission.

I'd suggest this:

* You decide if you'd like me to change the behat test to "test as admin" or to "test as user with permission to view course without being enrolled" and I'll see to it
* You decide if I should add another issue and/or behat test for the above thing with the "Einschreibeoptionen" placement (or even if you agree with me at all regarding this)

What do you think?